model/tensorflow_hub/setup.py

```diff
@@ -0,0 +1,93 @@
+import os
+import sys
+import ast
+from io import open
+
+from setuptools import find_packages, setup
+
+self_path = os.path.dirname(os.path.realpath(__file__))
+
+with open(
+    os.path.join(self_path, "dffml_model_tensorflow_hub", "version.py"), "r"
+) as f:
+    for line in f:
+        if line.startswith("VERSION"):
+            version = ast.literal_eval(line.strip().split("=")[-1].strip())
+            break
+
+with open(os.path.join(self_path, "README.md"), "r", encoding="utf-8") as f:
+    readme = f.read()
+
+INSTALL_REQUIRES = (
+    ["tensorflow>=2.0.0", "tensorflow-hub>=0.6.0", "pandas>=0.25.0",]
```

Comment:
I looks like `numpy` is also imported directly,

```suggestion
    ["tensorflow>=2.0.0", "tensorflow-hub>=0.6.0", "pandas>=0.25.0", "numpy>=1.16.4"]
```